src/Integration/TeamExplorer/ConnectSectionView.xaml

```diff
@@ -138,7 +139,7 @@
                             <Grid.InputBindings>
                                 <MouseBinding Command="{Binding Path=DataContext.BindCommand,
                                                                 RelativeSource={RelativeSource AncestorType=UserControl}}"
-                                              CommandParameter="{Binding .}"
+                                              CommandParameter="{Binding ., Converter={StaticResource ProjectViewModelToBindingArgsConverter}}"
```

Comment:
Some time ago I refactored the binding command to use strongly-typed arguments to make the binding code easier to understand. However, I failed to update this command binding.

The command now expects a parameter of type _BindCommandArgs_, but here the command is bound to a _ProjectViewModel_. Fortunately, the project view model contains all of the information needed to create a _BindCommandArgs_, so adding a simple converter is enough.